src/wallet.js

```diff
@@ -1097,6 +1116,16 @@ const wallet = {
     storage.setItem('wallet:closed', true);
   },
 
+  /*
+   * Set in storage as not closed
+   *
+   * @memberof Wallet
+   * @inner
+   */
+  afterOpen() {
```

Comment:
I don't get it. When is it called? What does `'wallet:closed'` mean?

Reply:
It's used to indicate if Hathor Wallet was closed, as we lock it. So next time user opens it, it will ask for pin.

Reply:
Ok. Thanks!